packages/react-components/src/templates/TeamProfileHeader.tsx

```diff
@@ -178,37 +148,10 @@ const TeamProfileHeader: React.FC<TeamProfileHeaderProps> = ({
       <section
         css={canCreateUpdate ? createSectionStyles : contactSectionStyles}
       >
-        <div css={membersContainerStyles}>
-          <ul css={membersListStyles}>
-            {members
-              .slice(0, MAX_MEMBER_AVATARS)
-              .map(({ id: memberId, avatarUrl, firstName, lastName }, i) => (
-                <li
-                  key={memberId}
-                  css={[listItemStyles, { left: `-${i * 3}px` }]}
-                >
-                  <Anchor
-                    href={network({}).users({}).user({ userId: memberId }).$}
-                  >
-                    <Avatar
-                      firstName={firstName}
-                      lastName={lastName}
-                      imageUrl={avatarUrl}
-                    />
-                  </Anchor>
-                </li>
-              ))}
-            <li css={extraUsersStyles}>
-              {members.length > MAX_MEMBER_AVATARS && (
-                <Anchor href={`${route.about({}).$}#${teamListElementId}`}>
-                  <Avatar
-                    placeholder={`+${members.length - MAX_MEMBER_AVATARS}`}
-                  />
-                </Anchor>
-              )}
-            </li>
-          </ul>
-        </div>
+        <MembersAvatars
```

Comment:
When i saw the MembersAvatars component i was worried this had been duplicated. Good work 👍